aws_quickstart/datadog_agentless_scanning.yaml

```diff
@@ -659,7 +667,14 @@ Resources:
           - Sid: EC2AssumeRole
             Effect: Allow
             Principal:
-              AWS: !GetAtt 'ScannerInstanceRole.Arn'
+              AWS: '*'
+            Condition:
+              ArnLike:
+                'aws:PrincipalArn': !GetAtt 'ScannerInstanceRole.Arn'
```

Comment:
`ScannerInstanceRole.Arn` is an actual ARN (no wildcard), right? So this will still only allow that specific ARN, which does not seem to be the intention.

Reply:
It is. The trust relationship of the delegate role still explicitly list the scanners that are allowed to assume the role. In this single-account case `'ScannerInstanceRole.Arn'`

Reply:
In that case what is the benefit of adding the `aws:PrincipalOrgID` condition? 😕